scripts/utils-wolfssl.sh

```diff
@@ -59,14 +67,25 @@ install_wolfssl() {
 
     if [ ! -d ${WOLFSSL_INSTALL_DIR} ]; then
         printf "\tConfigure wolfSSL ${WOLFSSL_TAG} ... "
-        if [ -z "$WOLFSSL_CONFIG_OPTS" ]; then
-            WOLFSSL_CONFIG_OPTS='--enable-all-crypto --with-eccminsz=192 --with-max-ecc-bits=1024 --enable-opensslcoexist --enable-sha'
-            WOLFSSL_CONFIG_CFLAGS="-I${OPENSSL_INSTALL_DIR}/include -DWC_RSA_NO_PADDING -DWOLFSSL_PUBLIC_MP -DHAVE_PUBLIC_FFDHE -DHAVE_FFDHE_6144 -DHAVE_FFDHE_8192 -DWOLFSSL_PSS_LONG_SALT -DWOLFSSL_PSS_SALT_LEN_DISCOVER"
-        fi
 
         ./autogen.sh >>$LOG_FILE 2>&1
-        ./configure ${WOLFSSL_CONFIG_OPTS} CFLAGS="${WOLFSSL_CONFIG_CFLAGS}" -prefix=${WOLFSSL_INSTALL_DIR} >>$LOG_FILE 2>&1
-        if [ $? != 0 ]; then
+        CONF_ARGS="-prefix=${WOLFSSL_INSTALL_DIR}"
+
+        if [ "$WOLFPROV_DEBUG" = "1" ]; then
+            CONF_ARGS+=" --enable-debug"
+        fi
+        if [ "$WOLFSSL_ISFIPS" = "1" ]; then
+            CONF_ARGS+=" --enable-fips=ready"
+            if [ ! -e "XXX-fips-test" ]; then
+                ./fips-check.sh keep nomakecheck fips-ready >>$LOG_FILE 2>&1
+                $(cd XXX-fips-test && ./autogen.sh && ./configure ${CONF_ARGS} ${WOLFSSL_CONFIG_OPTS} CFLAGS="${WOLFSSL_CONFIG_CFLAGS}" && make && ./fips-hash.sh) >>$LOG_FILE 2>&1
```

Comment:
whoops, this isn't supposed to be a `$(`, just a `(`.  credit to `shellcheck`:
```
In scripts/utils-wolfssl.sh line 81:
                $(cd XXX-fips-test && ./autogen.sh && ./configure ${CONF_ARGS} ${WOLFSSL_CONFIG_OPTS} CFLAGS="${WOLFSSL_CONFIG_CFLAGS}" && make && ./fips-hash.sh) >>$LOG_FILE 2>&1
                ^-- SC2091 (warning): Remove surrounding $() to avoid executing output (or use eval if intentional).
```